stellarphot/gui_tools/fits_opener.py

```diff
@@ -128,3 +128,4 @@ def set_file(self, file, directory=None):
 
         self._fc.reset(directory, file)
         self._fc._apply_selection()
+        self._fc._value = self._fc.selected
```

Comment:
I am a little wary of using "protected" attributes/methods of a class... I have done it, but I am wary of this.

Reply:
I had the same reservations, but figured it could wait....